site/docs/supported-models/_components/llm-models-table/models.ts

```diff
@@ -442,11 +439,8 @@ export const LLM_MODELS: LLMModelType[] = [
           'https://huggingface.co/internlm/internlm2-7b',
           'https://huggingface.co/internlm/internlm2-chat-20b',
           'https://huggingface.co/internlm/internlm2-20b',
-          'https://huggingface.co/internlm/internlm2_5-chat-1_8b',
```

Comment:
could you please replace instead :
https://huggingface.co/internlm/internlm2_5-1_8b-chat
https://huggingface.co/internlm/internlm2_5-7b-chat
https://huggingface.co/internlm/internlm2_5-20b-chat

Reply:
Replaced